diff --git a/include/configs/imx8mm_var_dart.h b/include/configs/imx8mm_var_dart.h
index f0ffcf9..e0cae0e 100644
--- a/include/configs/imx8mm_var_dart.h
+++ b/include/configs/imx8mm_var_dart.h
@@ -65,8 +65,8 @@
 /* Flat Device Tree Definitions */
 #define CONFIG_OF_BOARD_SETUP
 
-#undef CONFIG_CMD_EXPORTENV
-#undef CONFIG_CMD_IMPORTENV
+#define CONFIG_CMD_EXPORTENV    1
+#define CONFIG_CMD_IMPORTENV    1
 #undef CONFIG_CMD_IMLS
 
 #undef CONFIG_CMD_CRC32
@@ -101,7 +101,7 @@
 /* Initial environment variables */
 #define CONFIG_EXTRA_ENV_SETTINGS		\
 	CONFIG_MFG_ENV_SETTINGS \
-	"bootdir=/boot\0"	\
+	"bootdir=\0"	\
 	"script=boot.scr\0" \
 	"image=Image.gz\0" \
 	"console=ttymxc0,115200 earlycon=ec_imx6q,0x30860000,115200\0" \
